components/brave_rewards/resources/ui/components/listToken/index.tsx

```diff
@@ -24,19 +26,25 @@ export default class ListToken extends React.PureComponent<Props, {}> {
   }
 
   render () {
-    const { id, title, value, converted, isNegative, size, color, border, testId } = this.props
+    const { id, title, value, converted, isNegative, size, color, border, testId, link } = this.props
 
     return (
       <StyledWrapper id={id} border={border} data-test-id={testId}>
         <StyledTitle>{title}</StyledTitle>
         <StyledContentWrapper>
-          <Tokens
-            value={value}
-            converted={converted}
-            isNegative={isNegative}
-            size={size}
-            color={color}
-          />
+          {
+            link && link.length > 0
```

Comment:
I think that testing for `link.length > 0` is redundant when you already test `link`.